kolla-ansible/ansible/triliovault_site.yml

```diff
@@ -14,17 +14,20 @@
         tasks_from: loadbalancer
       when:
         - inventory_hostname in groups['triliovault-datamover-api']
+        - inventory_hostname in groups['triliovault-wlm-api']
         - enable_haproxy | bool
         - enable_triliovault | bool
         - kolla_action in ['deploy', 'reconfigure', 'upgrade', 'config']
-
+    
 - name: Apply role triliovault
   gather_facts: false
   hosts:
     - triliovault-datamover-api
     - triliovault-datamover
-    - triliovault-workloadmgr
-    - triliovault-workloadmgr-api
+    - triliovault-wlm-api
+    - triliovault_wlm_workloads
```

Comment:
Use hyphen instead of underscore character in host name of triliovault-wlm-workloads.